src/accounts/nuls2.ts

```diff
@@ -132,10 +139,93 @@ export async function ImportAccountFromPrivateKey(
     privateKey: string,
     { chain_id = 1, prefix = "NULS" }: NULS2ImportConfig = { chain_id: 1, prefix: "NULS" },
 ): Promise<NULS2Account> {
-    const pub = privateKeyToPublicKey(Buffer.from(privateKey, "hex"));
+    const pub = secp256k1.publicKeyCreate(Buffer.from(privateKey, "hex"));
     const publicKey = Buffer.from(pub).toString("hex");
 
     const hash = publicKeyToHash(pub, { chain_id: chain_id });
     const address = addressFromHash(hash, prefix);
     return new NULS2Account(address, publicKey, privateKey);
 }
+
+/********************************
+ * NULS2 TOOLS METHODS
+ ********************************/
+
```

Comment:
I have mixed feeling toward those tools. 
I get we should provide helpers on the NULS2 chain, but all the other accounts only exports methods to instantiate the Account class. 

I propose we either provide an `utils` file alongside the accounts, with helpers and typedefs, or use those functions as static methods on the NULSAccount class. What do you think ?

Reply:
Changed made in: https://github.com/aleph-im/aleph-sdk-ts/pull/43/commits/e20d579ea6f8e641b4398c0bd720f6538dba3b17

As NULS2 is the only chain that needs those tooling functions, making them static in the NULS2Account class makes more sense for me even if it makes the class slightly more blurry.